package.json

```diff
@@ -12,7 +12,7 @@
     "watch-views": "nodemon --watch server/views -e html,njk -x npm run copy-views",
     "watch-node": "DEBUG=gov-starter-server* nodemon --watch dist/ dist/server.js | bunyan -o short",
     "watch-sass": "npm run compile-sass -- --watch",
-    "build": "npm run compile-sass && tsc && npm run copy-views",
+    "build": "rm -rf dist && npm run compile-sass && tsc && npm run copy-views",
     "start": "node $NODE_OPTIONS -r dotenv/config dist/server.js | bunyan -o short",
```

Comment:
Maybe a `clean` target for the `rm -rf dist` ? Makes it optional with `npm run clean build`. Not sure it makes a difference though.

Reply:
Will have a look at this (added this from DPS when i ran into a caching issues)

Reply:
Actually create and vary a license have this as well - though they also have a clean script that clears down the node modules folder